certs/service_test.go

```diff
@@ -65,7 +65,7 @@ func newService(tokens map[string]string) (certs.Service, error) {
 
 	auth := thmocks.NewAuthService(tokens)
 	config := mfsdk.Config{
-		CertsURL: server.URL,
+		ThingsURL: server.URL,
```

Comment:
What is this `server`? Is it Bootstrap server? If yes - why we are giving this URL to Things service configuration - in realistic deployment these will be two completely different URLs.

Reply:
`server` here is http server serving API calls to things service

Reply:
Yes, serving HTTP API of a *Bootstrap* service, i.e. it is a Bootstrap service `httptest` server. This is why I am saying that this is fundamentally wrong.

Reply:
not bootstrap, but things